scala/private/phases/phase_write_executable.bzl

```diff
@@ -73,3 +79,107 @@ def _phase_write_executable(
         wrapper,
         use_jacoco,
     )
+
+def _write_executable(ctx, executable, rjars, main_class, jvm_flags, wrapper, use_jacoco):
```

Comment:
Combine `_phase_write_executable` and `_write_executable`?

Reply:
fixed